src/main/java/com/opencart/pages/HeaderPage.java

```diff
@@ -51,6 +51,15 @@
     @FindBy(css = "a[title = 'Shopping Cart'] span")
     private WebElement shoppingCartSpan;
 
+    @FindBy(css = "span#cart-total")
+    private WebElement shoppingCartDropDownInfo;
+
+    @FindBy(css = "p.text-center")
+    private WebElement shoppingCartDropDownMessage;
+
+    @FindBy(css = "button.btn.btn-inverse.btn-block.btn-lg.dropdown-toggle")
```

Comment:
Before making the commit try to use shortcuts to format your code, so all the extra spaces will be removed automatically.